pkg/webhook/policy.go

```diff
@@ -18,45 +18,30 @@ import (
 	authenticationv1 "k8s.io/api/authentication/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	k8sruntime "k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/serializer"
-	"k8s.io/apimachinery/pkg/types"
+	// k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	// "k8s.io/apimachinery/pkg/runtime/serializer"
+	// "k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
-	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	// logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
```

Comment:
We can drop commented-out imports.

Also, I don't understand the comment:

`// can get rid of this, gk-wh replaces it`. There is still a dependency on the webhook package here.

Reply:
That was an old comment, I forgot to remove it after it became irrelevant.

Reply:
Done.